### PR TITLE
Pubsub

### PR DESCRIPTION
**Pull request by JamesTFarrington**
Made some changes to hear_from_backend. If you could take a look at the last too commits that would be great.
Anything Important should be visible here:
https://github.com/jMyles/tiempo/commit/a0e7785a227c5f94a10339ae9c5a9ad655928767

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,9 @@ Point your browser to 127.0.0.1:4050/tiempo_kiosk
 #### To use with Django/Hendrix
 
 *  in settings.py
-  * `TIEMPO_THREAD_CONFIG = [('prioirty1'), ('priority1, 'downstairs', 'upstairs')]`
+  * `TIEMPO_THREAD_CONFIG = [('prioirty1'), ('priority1', 'downstairs', 'upstairs')]`
     * each entry in the list specifies which queues that thread should check for work 
     * the above example will have two both checking "priority1" and "downstairs" and "upstairs" on only one process
-  * 'INSTALLED_APPS' += 'tiempo.contrib.django_app'
+  * 'INSTALLED_APPS' += 'tiempo.contrib.django_APP'
```
Comment (jMyles) [inline README.md:26]: Why the caps here?

File: tiempo/conn.py [hunk 2]
```diff
@@ -20,10 +21,15 @@ def subscribe_to_backend_notifications(db=REDIS_QUEUE_DB):
 
 def hear_from_backend():
     events = []
-    while True:
-        message = NOTIFY_PUBSUB.parse_response(block=False)
+    def parse_backend():
```
Comment (jMyles) [inline tiempo/conn.py:24]: It's not immediately obvious to me what the advantage is of adding this closure. Maybe a comment?

File: README.md (hunk 1 above)
Comment (JamesTFarrington) [inline README.md:26]: I'm not sure how that happened actually.

**Comment by JamesTFarrington** [comment]
hear_from_backend is being called from inside of a loop. Each time it is called it creates a new events list and starts a while loop. With a closure it uses the same events list each time it's called, and we can remove the while loop because it's being called repeatedly from inside a loop.

**Comment by jMyles** [comment]
Subsumed. https://github.com/hangarunderground/tiempo/tree/2.0-thunderdome

**Pull request closed without merging**